tests/src/Unit/EntityType/EntityBaseFieldInfoAlterEventTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\Tests\hook_event_dispatcher\Unit\EntityType;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\hook_event_dispatcher\HookEventDispatcherInterface;
+use Drupal\Tests\hook_event_dispatcher\Unit\HookEventDispatcherManagerSpy;
+use Drupal\Tests\UnitTestCase;
+
+class EntityBaseFieldInfoAlterEventTest extends UnitTestCase {
+
+  /**
+   * The manager.
+   *
+   * @var \Drupal\Tests\hook_event_dispatcher\Unit\HookEventDispatcherManagerSpy
+   */
+  private $manager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    $builder = new ContainerBuilder();
+    $this->manager = new HookEventDispatcherManagerSpy();
+    $builder->set('hook_event_dispatcher.manager', $this->manager);
+    $builder->compile();
+    \Drupal::setContainer($builder);
+  }
+
+  /**
+   * Test the EntityBaseFieldInfoEvent.
+   */
+  public function testEntityBaseFieldInfoEvent() {
```

Comment:
This test misses the real life usage of this event, please add an extra field to the fields after the event is fired. You can look up how I do this with the HookEventDispatcherManagerSpy in other tests. You could have a look at the `ViewEventTest` for instance.